people/pau-giner.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Pau Giner, Designer
+name: Pau Giner
+github: @pauginer
```

Comment:
Small note here: front matter doesn't like @ signs as the symbol in the value. Just having `pauginer` here would work.